packages/tests/package.json

```diff
@@ -13,6 +13,7 @@
     "@fastify/busboy": "^2.0.0",
     "@fastify/websocket": "^7.1.2",
     "@miniflare/core": "^2.9.0",
+    "@robolex/sure": "^0.11.2",
     "@tanstack/react-query": "^5.0.0",
```

Comment:
Please don't add this library as an officially supported one before it reaches `1.0.0`, I don't want us to maintain random breaking changes

If you want, just remove it from the test suite and have it in the docs without using twoslash

Reply:
Got it, removed the tests and twoslash